### PR TITLE
adding some missing acquisition parameters

### PR DESCRIPTION
**Pull request by fonsecareyna82**

### CODE REVIEW

**Comment by fonsecareyna82** [comment]
Closes #391

**Comment by azazellochg** [comment]
Hi @fonsecareyna82 , could we remove magnification please?

**Comment by fonsecareyna82** [comment]
Hi @azazellochg , let me check that, because I think it is being used somewhere.

**Comment by azazellochg** [comment]
dynamo tbl?

**Comment by azazellochg** [comment]
I doubt its value means anything since the pixel size should be used instead... but yes, good to check

File: tomo/protocols/protocol_base.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 # **************************************************************************
 
 import pyworkflow as pw
+from pyworkflow.protocol import IntParam
```
Comment (azazellochg) [inline tomo/protocols/protocol_base.py:28]: I guess it can be imported on the line below

**Review by azazellochg — APPROVED** [review]

**Pull request merged**